learn/advanced/updating.md

```diff
@@ -130,7 +130,7 @@ Start by using the [get displayed attributes endpoint](/reference/api/displayed_
 ```bash
 # whenever you see :index_uid, replace it with your index's unique id
```

Comment:
Commenting only here, but all these code blocks have comments that must also be updated.

```suggestion
# whenever you see {index_uid}, replace it with your index's unique id
```

Reply:
Thanks Gui!